python/test/unit/language/test_conversions.py

```diff
@@ -259,6 +258,9 @@ def test_typeconvert_upcast(src_dtype, dst_dtype):
     if src_dtype == 'float8e4nv' and torch.cuda.get_device_capability(0) < (9, 0):
         pytest.skip("float8e4nv upcast tests only supported on compute capability 9.0+")
 
+    if src_dtype == 'float8e4nv' and torch.version.hip is not None:
+        pytest.skip("float8e4nv upcast tests not supported on ROCm")
```

Comment:
nit: can you merge it with the check above.

Reply:
Done.